shuup/admin/utils/wizard.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.conf import settings
+
+from shuup import configuration
+from shuup.core.models import Shop, ShopStatus
+
+
+def setup_wizard_complete():
+    """
+    Check if shop wizard should be run.
+
+    :return: whether setup wizard needs to be run
+    :rtype: boolean
+    """
+    if getattr(settings, "SHUUP_ENABLE_MULTIPLE_SHOPS", False):
+        # setup wizard is only applicable in single shop mode
+        return True
+    shop = Shop.objects.first()
```

Comment:
Something else should be checked here too. Maybe that there is at least one enabled payment and shipping method. Also theme should be active. Maybe it could even tell a reason why the wizard pop up.